Objective-C/CBLFullTextIndexConfiguration.h

```diff
@@ -40,13 +40,27 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, readonly, nullable) NSString* language;
 
+/**
+ A predicate expression defining conditions for indexing documents.
+ Only documents satisfying the predicate are included, enabling partial indexes.
+ */
+@property (nonatomic, readonly, nullable) NSString* where;
+
 /**
  Constructor for creating a full-text index by using an array of expression strings
  */
 - (instancetype) initWithExpression: (NSArray<NSString*>*)expressions
                       ignoreAccents: (BOOL)ignoreAccents
                            language: (NSString* __nullable)language;
 
+/**
+ Constructor for creating a partial full-text index by using an array of expression strings
+ */
+- (instancetype) initWithExpression: (NSArray<NSString*>*)expressions
+                      ignoreAccents: (BOOL)ignoreAccents
+                           language: (NSString* __nullable)language
+                              where: (nullable NSString*)where;
```

Comment:
Seem like where and language are using a different way to make nullable. Can you check which method is the preferred or right one?